carbon-design-system/carbon
# Tag: fix accessibility violation

Closes #13706 

By removing the `aria-label` from the div tag the violation goes away.
I tested with the voice over from MacOS and the readability still works as expected.

#### Testing / Reviewing

To reproduce the error or check the fix, you can follow these steps:

- Open the `Tag` storybook;
- In Controls section set the filter prop to true;
- Open the component itself in a new tab;
- Then run the IBM Equal Access Accessibility Checker;
- The violation should be gone.
